src/python/bot/tasks/fuzz_task.py

```diff
@@ -1601,9 +1613,9 @@ def run(self):
       # testcase is analyzed.
       android.device.initialize_device()
 
-    # Transform tests.Crash into fuzz_task.Crash.
+    # Transform crashes into fuzz_task.Crash.
     # And filter the crashes (e.g. removing errorneous crashes).
-    crashes = [Crash(crash) for crash in crashes]
+    crashes = filter(None, [crash_constructor(crash) for crash in crashes])
```

Comment:
I don't understand what's the purpose of `filter(None, ...)`. Could you please explain?
It's not trying to get rid of Nones is it? (I don't think it works for that purpose).
Is this a common idiom?

Reply:
It is indeed to get rid of Nones, and I believe this is a common idiom. Passing None as the first argument means we use the identity function.

Reply:
ACK